qe_gluster_cluster.yml

```diff
@@ -0,0 +1,12 @@
+---
+#
+# ansible-gluster libraries required
+# https://github.com/dahorak/ansible-gluster.git or
+# https://github.com/loadtheaccumulator/ansible-gluster.git
+# export ANSIBLE_LIBRARY=../ansible-gluster/
```

Comment:
We need to add this into qe server setup role. But not in this pull request or this branch.